src/settings/bool-setting/bool-setting.ts

```diff
@@ -38,18 +36,28 @@ export class UIPBoolSetting extends UIPSetting {
     return $field;
   }
 
-  protected connectedCallback() {
-    super.connectedCallback();
+  @memoize()
+  protected get $label(): HTMLLabelElement {
+    const $label = document.createElement('label');
+    $label.innerText = this.label;
+    $label.append(this.$field);
+    return $label;
+  }
 
-    const label = document.createElement('label');
-    label.innerText = this.label;
-    label.appendChild(this.$field);
+  @memoize()
+  protected get $inconsistencyMarker(): HTMLElement {
+    const marker = document.createElement('div');
+    marker.classList.add('inconsistency-marker');
+    return marker;
+  }
 
-    this.innerHTML = '';
-    this.appendChild(label);
+  protected connectedCallback() {
+    super.connectedCallback();
+    this.$label.remove();
```

Comment:
There is no need to do that
The active DOM behavior is the following: 
 - one Element can be placed just in one place in time
 - in case we add the Element to the new position, it will be automatically removed from its previous position